arbitrum-docs/devs-how-tos/how-to-use-oracles.mdx

```diff
@@ -0,0 +1,254 @@
+---
+title: "How to use Oracles in Arbitrum"
+sidebar_label: Use Oracles in Arbitrum
+description: Learn how to integrate Oracles into your dapp
+reader-audience: developers who want to build on Ethereum/Arbitrum
+content-type: how-to
+---
+
+import PublicPreviewBannerPartial from '../partials/_public-preview-banner-partial.md';
+
+<PublicPreviewBannerPartial />
+
+In this how-to, you’ll learn how to incorporate popular Oracle services into your Arbitrum dApps. Specifically, each section will provide step-by-step instructions on how to obtain current asset prices from oracles, helping you seamlessly integrate these services.
+
+This guide assumes that you're familiar with the concept of [Oracles](../for-devs/concepts/oracles.mdx) and have a basic understanding of smart contracts and blockchain development. If you’re new to blockchain development, consider going through our [Quickstart: Build a dApp with Arbitrum (Solidity, Hardhat)](../for-devs/quickstart-solidity-hardhat) before proceeding.
+
+## Chainlink
+
+[Chainlink](https://chain.link/) is a widely-recognized Web3 services platform that specializes in decentralized oracle networks. It allows developers to create versatile Web3 applications that can effortlessly connect to real-world data and off-chain computation on Arbitrum and many other blockchain networks. Chainlink offers a variety of data feeds, including asset prices, weather data, random number generation, and many more.
```

Comment:
```suggestion
[Chainlink](https://chain.link/) is a widely-recognized Web3 services platform that specializes in decentralized oracle networks. It lets you build Ethereum and Arbitrum dApps that connect to a variety of off-chain data feeds and APIs, including those that provide asset prices, weather data, random number generation, and more.
```

Definitely feel free to discard/partially incorporate if this edit is too "destructive".